manual/src/installation_features.md

```diff
@@ -19,22 +19,23 @@ features = ["async-std-runtime"]
 ```
 
 ## Enabling the sync API
-The driver also provides a blocking sync API. To enable this, add the `"sync"` feature to your `Cargo.toml`:
+The driver also provides a blocking sync API. To enable this, add the `"sync"` or `"tokio-sync"` feature to your `Cargo.toml`:
 ```toml
 [dependencies.mongodb]
 version = "2.1.0"
```

Comment:
sort of unrelated: this should be `2.2.0`. We'll probably want to update the release checklist to include the manual as another place needing updating when doing releases.